packages/syft/src/syft/core/tensor/autodp/gamma_tensor.py

```diff
@@ -1994,6 +1994,88 @@ def __le__(self, other: Any) -> GammaTensor:
             sources=output_state,
         )
 
+    def __abs__(self) -> GammaTensor:
+
+        output_state = dict()
+        output_state[self.id] = self
+
+        data = self.child
+        child = np.abs(data)
```

Comment:
I think we could call data.__abs__() itself, because during SMPC the child is a FixedPrecisionTensor instead of Numpy array.